client/lib/analytics/ad-tracking.js

```diff
@@ -1019,15 +1016,10 @@ export function recordAddToCart( cartItem ) {
 	// DCM Floodlight
 
 	if ( isFloodlightEnabled ) {
-		const params = {
-			type: 'wordp0',
-			cat: 'addto0',
-			u2: cartItem.product_name,
-			ord: floodlightCacheBuster(), // "Counting method" = "Counter/Standard" requires "ord" to be a random number
-			// num: not required
-		};
-		debug( 'recordAddToCart: [Floodlight]', params );
-		recordParamsInFloodlight( params );
+		debug( 'recordAddToCart: [Floodlight]' );
+		recordParamsInFloodlightGtag( {
+			send_to: 'DC-6355556/wordp0/addto0+standard',
```

Comment:
Missing cart item (product name) in `u2`?

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/32205/commits/e4b6573280e7ead143d902242e40415cf46487d8

Reply:
Thanks!